var/spack/repos/builtin/packages/py-dlio-profiler-py/package.py

```diff
@@ -15,27 +15,47 @@ class PyDlioProfilerPy(PythonPackage):
 
     license("MIT")
 
-    version("develop", branch="dev")
-    version("master", branch="master")
-    version("0.0.5", tag="v0.0.5", commit="08f1a43c67c8dbb458d547020674c86118c9742e")
-    version("0.0.4", tag="v0.0.4", commit="f9ba207f4c3e3789eb7759653a94013e6b76c91c")
-    version("0.0.3", tag="v0.0.3", commit="531f4475cf03312e121c78bf644445882b51ad57")
-    version("0.0.2", tag="v0.0.2", commit="b72144abf1499e03d1db87ef51e780633e9e9533")
-    version("0.0.1", tag="v0.0.1", commit="28affe716211315dd6936ddc8e25ce6c43cdf491")
+    version(
+        "0.0.7", tag="v0.0.7", commit="e47ec476b58e14157b807cbadb4187bd4fe811d9", deprecated=True
+    )
+    version(
+        "0.0.6", tag="v0.0.6", commit="3be111c973883387418ad96f63a18de63555c540", deprecated=True
+    )
```

Comment:
```suggestion
```

Reply:
So, I cannot remove these currently, as some users want to use 0.0.7. The migration to DFTracer would take longer for them. That's why I added the latest tag here: for the moment, they can use 0.0.7.